### ISSUE TITLE
CocoonJS plugin causes HTML exports to have a white background

### ISSUE THREAD

**Issue by SeanNoonan** [ludei/Construct-2-plugin#62]
When exporting a project to HTML5, the generated HTML file has a white background if the CocoonJS plugin is present in the project - it is usually black and can be configured via the HTML file after the export.

**Comment by LibelulaV** [comment]
Hello, 

And what about our exporter? In addition, does this mean you can't set the bg to black after the exportation? 

Regards

**Comment by SeanNoonan** [comment]
Yeah, it seems to be stuck as white if the plugin is present - I basically have to remove the plugin for exports that aren't for iOS purposes.
